vars/coverityPost.groovy

```diff
@@ -0,0 +1,53 @@
+// vars/coverityPost.groovy
+
+  /**
+   * coverityPost step method
+   *
+   * @param config Map of parameters passed
+   *
+   * config['condition']           Name of the post step to run. 
+   *                               Rrequired to be one of
+   *                               'success', or 'unsuccessful'.
+   *
+   * config['success_script']      Script to run on successful build.
+   *                               Default 'ci/coverity_success.sh'.
+   *
+   * config['unsuccessful_script'] Script to run if build is not successful.
+   *                               Default 'ci/coverity_unsuccessful.sh'
+   *
+   * config['coverity_tarball']    Tarball to archive for coverity.
+   *                               default: 'coverity/*.tgz'
+   */
+
+def call(Map config = [:]) {
+
+  if (config['condition'] == 'success') {
+    
```

Comment:
Tiny nit but extra/inconsistent whitespace here:
```suggestion    
```

Reply:
Fixed in next push.